packages/palette/src/utils/useCurrentTime.ts

```diff
@@ -0,0 +1,20 @@
+import { DateTime } from "luxon"
+import { useState } from "react"
+import { useInterval } from "./useInterval"
+
+/**
+ * useCurrentTime
+ */
+export function useCurrentTime(currentTime) {
```

Comment:
Do the existing tests cover this hook pretty well? If not, I think tests against `useCurrentTime` directly would be valuable.